scripts/plotting/polar_map.py

```diff
@@ -224,36 +229,42 @@ def polar_map(adfobj):
-                            shfig = make_polar_plot(mseasons[s], oseasons[s], dseasons[s], hemisphere="SH", **vres)
-
-                            # Save files
-                            nhfig.savefig(nh_plot_name, bbox_inches='tight', dpi=300)
-                            shfig.savefig(sh_plot_name, bbox_inches='tight', dpi=300)
-
-                            # Close figures to avoid memory issues:
-                            plt.close(nhfig)
-                            plt.close(shfig)
+                            # If redo_plot set to True: remove old plot, if it already exists:
+                            for plot_name in [nh_plot_name, sh_plot_name]:
+                                print(plot_name)
+
+                                if (not redo_plot) and plot_name.is_file():
+                                    continue
+                                elif (redo_plot) and plot_name.is_file():
+                                    plot_name.unlink()
+                            
+                                    #Create new plot:
+                                    # NOTE: send vres as kwarg dictionary.  --> ONLY vres, not the full res
+                                    # This relies on `plot_map_and_save` knowing how to deal with the options
+                                    # currently knows how to handle:
+                                    #   colormap, contour_levels, diff_colormap, diff_contour_levels, tiString, tiFontSize, mpl
+                                    #   *Any other entries will be ignored.
+                                    # NOTE: If we were doing all the plotting here, we could use whatever we want from the provided YAML file.
+                                    
+                                    # pf.plot_map_and_save(plot_name, mseasons[s], oseasons[s], dseasons[s], **vres)
+                                    
+                                    nhfig = make_polar_plot(mseasons[s], oseasons[s], dseasons[s], hemisphere="NH", **vres)
+                                    shfig = make_polar_plot(mseasons[s], oseasons[s], dseasons[s], hemisphere="SH", **vres)
+                                    
+                                    # Save files
+                                    nhfig.savefig(nh_plot_name, bbox_inches='tight', dpi=300)
+                                    shfig.savefig(sh_plot_name, bbox_inches='tight', dpi=300)
+                                    
+                                    # Close figures to avoid memory issues:
+                                    plt.close(nhfig)
+                                    plt.close(shfig)
 
                     else: #mdata dimensions check
-                        print("\t - skipping polar map for {} as it doesn't have only lat/lon dims.".format(var))
+                        print("\t   INFO: Skipping polar map for {} as it doesn't have only lat/lon dims.".format(var))
```

Comment:
I know you didn't write it this way, but can you convert this  to an f-string:

```suggestion
                        print(f"\t   INFO: Skipping polar map for {var} as it doesn't have only lat/lon dims.")
```